tools/ctl/ctl.go

```diff
@@ -101,10 +102,7 @@ var tui = &cobra.Command{
 			tctx = context.WithValue(tctx, rebuild.DebugStoreID, "file://"+localfiles.AssetsPath())
 		} else {
 			if *debugStorage != "" {
-				u, err := url.Parse(*debugStorage)
-				if err != nil {
-					log.Fatal(errors.Wrap(err, "parsing --debug-storage as url"))
-				}
+				u := urlx.MustParse(*debugStorage)
```

Comment:
lets revert these. panic's output isn't desirable in the CLI as it includes a full stacktrace. log.Fatal only outputs the error content.